src/Engine/Entity/Entity.hpp

```diff
@@ -127,7 +127,7 @@ class Entity {
         glm::vec3 GetWorldPosition() const;
         
         /// Name of the entity.
-        std::string name;
+        std::string name = "";
```

Comment:
This value is immediately overwritten in all cases. Would be better to alter constructor to use initializer list.